tap_zendesk/streams.py

```diff
@@ -16,6 +18,7 @@
 
 DEFAULT_PAGE_SIZE = 100
 REQUEST_TIMEOUT = 300
+DEFAULT_BATCH_SIZE = 700
```

Comment:
can this be changed?
Is this the lowest or highest possble value?

Reply:
This is a standard limit. However, we are setting its run time based on the API limit itself over [here](https://github.com/singer-io/tap-zendesk/pull/150/files#diff-89887ca52395ce152e9723c5595655edcd41793204060336e88ea9805c274433R363).